app/api/ddi-index-api/persons.js

```diff
@@ -0,0 +1,56 @@
+const { get } = require('./base')
+
+const personsEndpoint = 'persons'
+
+const options = {
+  json: true
+}
+
+/**
+ * @typedef GetPersonsFilterOptions
+ * @property {string} firstName
+ * @property {string} lastName
+ * @property {string} dateOfBirth
+ */
+/**
+ * @typedef GetPersonsFilterKeys
+ * @type {'firstName'|'lastName'|'dateOfBirth'}
+ */
+/**
+ * @param {{ firstName: string; lastName: string; dateOfBirth?: string; }} filter
+ * @returns {Promise<Person[]>}
+ */
+const getPersons = async (filter) => {
+  const urlParams = []
+  /**
+   * @type {GetPersonsFilterKeys[]}
+   */
+  const requiredParams = ['firstName', 'lastName']
+  /**
+   * @type {GetPersonsFilterKeys[]}
+   */
```

Comment:
I don;t want to cause you extra work but Joi can validate GET params (see https://www.npmjs.com/package/express-joi-validation#validation-ordering). Probably leave this as is but if we hit a more complex scenario, it might be more consistent to use Joi

Reply:
aha, cool, will do this then